### PR TITLE
[LOOP-349] Convert to Loop plugin

### PR DESCRIPTION
**Pull request by darinkrauss**

### CODE REVIEW

**Review by nhamming — DISMISSED** [review]
LGTM

File: NightscoutServiceKitPlugin/Info.plist [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>FMWK</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleVersion</key>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>com.loopkit.Loop.ServiceDisplayName</key>
+	<string>Nightscout Service</string>
```
Comment (ps2) [inline NightscoutServiceKitPlugin/Info.plist:22]: nit: I think "Service" is not needed here.

**Review by ps2 — DISMISSED** [review]
LGTM!

**Review by nhamming — COMMENTED** [review]
+1

**Review by ps2 — APPROVED** [review]
LGTM

**Pull request merged**